src/Microsoft.AspNet.Mvc.ModelBinding/BindAttribute.cs

```diff
@@ -110,11 +110,9 @@ string IModelNameProvider.Name
                 if (!initialized)
                 {
                     var services = context.OperationBindingContext.HttpContext.RequestServices;
-                    var activator = services.GetService<ITypeActivator>();
+                    var modelBinderActivator = services.GetRequiredService<IModelBinderActivator>();
 
-                    var provider = (IPropertyBindingPredicateProvider)activator.CreateInstance(
-                        services, 
-                        predicateProviderType);
+                    var provider = (IPropertyBindingPredicateProvider)modelBinderActivator.CreateInstance(services, predicateProviderType);
```

Comment:
The design of this could be pretty similar to that of typefilterattribute. Is `IModelBinderActivator` used elsewhere?